src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE [dbo].[Group]
+(
+    [GroupId] BIGINT NOT NULL CONSTRAINT PK_Group PRIMARY KEY IDENTITY,
+    [OwnerApplicationUserId]  BIGINT NOT NULL,
```

Comment:
@tarunsamanta2k20 please add a foreign key to [ApplicationUser].[ApplicationUserId]